wren-ai-service/src/providers/llm/openai.py

```diff
@@ -42,18 +42,20 @@ class OpenAILLMProvider(LLMProvider):
     def __init__(
         self,
         api_key: Secret = Secret.from_env_var("OPENAI_API_KEY"),
+        api_base: Secret = Secret.from_env_var("OPENAI_API_BASE"),
```

Comment:
can you also add a default api_base like the following, the same is the argument for generation_model?

`api_base: Secret = Secret.from_env_var("OPENAI_API_BASE") or OPENAI_API_BASE`

and the OPENAI_API_BASE should be `https://api.openai.com/v1`